book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
```

Comment:
- [ ] The **`Book`** class should have a method for adding rentals. Kindly add this. For example the method can take in two parameters (person and date) and return a Rental instance that takes in `date`, `person` and, `self` as parameters.